src/PostalRegistry.Api.Legacy/PostalInformation/Responses/PostalInformationLdesMetadata.cs

```diff
@@ -0,0 +1,110 @@
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using NodaTime;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PostalRegistry.Api.Legacy.PostalInformation.Responses
+{
+    public class PostalInformationLdesMetadata
+    {
+        public static Uri GetPageIdentifier(IConfiguration configuration, int page)
```

Comment:
Instead of depending on IConfiguration can you instead Do one of the following?
- if only one or two parameters are introduced use `string param1` or whatever the type/parameter is.
- use an `options` object which transforms your config to an object which can then be passed through.

Reply:
Look at this example too:
* https://github.com/Informatievlaanderen/organisation-registry/blob/main/src/OrganisationRegistry.ElasticSearch/ElasticSearchModule.cs#L12
* https://github.com/Informatievlaanderen/organisation-registry/blob/d142eb64126b395ab8e32a9c03bdb5b4d0151ba5/src/OrganisationRegistry.ElasticSearch/Configuration/ElasticSearchConfiguration.cs#L16
* https://github.com/Informatievlaanderen/organisation-registry/blob/main/src/OrganisationRegistry.ElasticSearch.Projections/appsettings.json#L25